nni/algorithms/compression/v2/pytorch/pruning/basic_pruner.py

```diff
@@ -165,27 +165,47 @@ class LevelPruner(BasicPruner):
         the non-zero value in tensor which create chance for better hardware acceleration.
 
         Note: If length of given balance_gran smaller than length of pruning tensor shape, it will be made up
-              in right align(such as example 1).
+        in right align(such as example 1).
+
+        Example 1::
+
+            operation: Linear
+            pruning tensor: weight
+            pruning tensor shape: [32, 32]
+            sparsity: 50%
+            balance_gran: [4]
+            pruning result: Weight tensor whose shape is [32, 32] will be split into 256 [1, 4] sub blocks.
+                            Each sub block will be pruned 2 values.
+
+        Example 2::
+
+            operation: Linear
+            pruning tensor: weight
+            pruning tensor shape: [64, 64]
+            sparsity: 25%
+            balance_gran: [32, 32]
+            pruning result: Weight tensor whose shape is [64, 64] will be split into 4 [32, 32] sub blocks.
+                            Each sub block will be pruned 256 values.
+    block_sparse_size : Union[int, List[int], None]
+        (experimental) By default, block_sparse_size is None, which means element-wise pruning with block size [1, 1, ..., 1].
```

Comment:
which means pruning granularity is single value.

Reply:
fixed, and split LevelPruner to three pruner